myapp/spec/models/task_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Task Model', type: :model do
+  describe 'validation' do
+    subject { proc { task.valid? } }
+
+    let!(:task) { FactoryBot.create(:task) }
+
+    describe 'title' do
+      it 'is not valid if it is blank.' do
+        task.title = ''
+        expect(task.valid?).to eq false
+      end
```

Comment:
条件系はcontextにしてネストするのが良さそうです！
↓参考資料
https://qiita.com/uchiko/items/d34c5d1298934252f58f
```suggestion
      context 'is blank' do
        it 'not valid' do
          task.title = ''
          expect(task.valid?).to eq false
        end
      end
```